horizon/transforms.py

```diff
@@ -78,21 +83,24 @@ def horizon_base_rgb(imgsz: int) -> A.Compose:
     )
 
 
-def horizon_augment_ir16bit(imgsz: int) -> A.Compose:
+def horizon_augment_ir16bit(imgsz: int, im_compression_prob:float) -> A.Compose:
     """
     Augmentations for 16-bit IR images.
 
     Args:
         imgsz (int): image size
+        im_compression_prob (float): Image compression propability 
     """
-    return A.Compose(
-        [
+
+    prefix = colorstr("albumentations ir16bit: ")
```

Comment:
same here, minor detail but if there is already some pattern going on, why changing it?